src/SelfService/Infrastructure/Api/ApiResourceFactory.cs

```diff
@@ -961,4 +984,85 @@ public TeamListApiResource Convert(List<Team> teams)
             )
         );
     }
+
+    public InvitationListApiResource Convert(IEnumerable<Invitation> invitations, string userId)
+    {
+        return new InvitationListApiResource(
+            items: invitations.Select(Convert).ToArray(),
+            links: new InvitationListApiResource.InvitationListLinks(
+                self: new ResourceLink(
+                    href: _linkGenerator.GetUriByAction(
+                        httpContext: HttpContext,
+                        controller: GetNameOf<InvitationController>(),
+                        action: nameof(InvitationController.GetActiveInvitations),
+                        values: new { userId }
+                    ) ?? "",
+                    rel: "self",
+                    allow: Allow.Get
+                )
+            )
+        );
+    }
+
+    public InvitationListApiResource Convert(IEnumerable<Invitation> invitations, string userId, string targetType)
```

Comment:
do we always call the converters just for `Convert`? It would be nice with a more descriptive name to understand what is converted from and to

Reply:
All our converters are called `Convert`.
Is it bad naming? Debatable.
Should we start renaming all these functions as part of this PR? I do not believe so, no.